utils/build.ps1

```diff
@@ -2818,14 +2818,12 @@ function Build-Foundation {
     -Bin $FoundationBinaryCache `
     -InstallTo $FoundationImage `
     -Platform $Platform `
-    -UseBuiltCompilers ASM,C,CXX,Swift `
+    -UseBuiltCompilers C,CXX,Swift `
     -SwiftSDK $SwiftSDK `
     -Defines @{
       BUILD_SHARED_LIBS = if ($Static) { "NO" } else { "YES" };
-      CMAKE_NINJA_FORCE_RESPONSE_FILE = "YES";
       CMAKE_STATIC_LIBRARY_PREFIX_Swift = "lib";
       CMAKE_Swift_FLAGS = $SwiftFlags;
-      ENABLE_TESTING = "NO";
```

Comment:
Oh, I guess this is now safe to remove because the testing support was removed upstream 😮‍💨